test/inductor/test_torchinductor.py

```diff
@@ -6038,6 +6036,30 @@ def fn(x):
                     assert same(_fn(x), compiled(x), equal_nan=True)
                     assert metrics.generated_cpp_vec_kernel_count == 1
 
+        @unittest.skipIf(
+            not codecache.valid_vec_isa_list(), "Does not support vectorization"
+        )
```

Comment:
Do we need to unify this code before **test_where_with_logical_op** case?

Reply:
They are two different things. This case only focuses on vectorization while `test_where_with_logical_op` covers both vectorization and scalar.